account_payment_mode/hooks.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def pre_init_hook(cr):
+    migrate_from_8(cr)
+
+
+def migrate_from_8(cr):
+    """If we're installed on a database which has the payment_mode table
+    from 8.0, move its table so that we use the already existing modes"""
+    cr.execute("SELECT 1 FROM pg_class WHERE relname = 'payment_mode'")
+    if not cr.fetchone():
+        return
+    cr.execute('ALTER TABLE payment_mode RENAME TO account_payment_mode')
```

Comment:
We should use openupgradelib method for renaming models, so that we don't lose for example possible attachments linked to these records (rare, but possible).